ansible/roles/data-products-deploy/defaults/main.yml

```diff
@@ -249,7 +259,7 @@ uci_postgres:
   user_identities_table_name: "identities"
 
 uci_encryption_secret_key: "{{uci_encryption_key_base64}}"
-uci_pdata_id: "{{ env_name}}.uci.sunbird"
```

Comment:
what should be the expected value of the pdata_id? `env_name` is having "sunbirddev" and `env` is having "dev".